### PR TITLE
Refactor awesome books projects and use es6

### PR DESCRIPTION
**Pull request by bizip**
**What does this PR do?**

- Refactor awesome books projects and use es6

**Description of Task to be completed**

- All tasks related to deployment are completed 

**How should this be manually tested?**

1. Clone this repository and switch to ft-logout
2. Install all required dependencies `npm install`
3. Open the HTML file with a live server

**Link to the live project**
[Awesome books](https://bizip.github.io/awesome_books_es6/)

- Due to many errors in the luxon.js library I have temporarily disabled the linter for javascript because it is an extra-large file with too many unfixable errors

### CODE REVIEW

File: modules/books.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+export default class Book {
+  constructor() {
+    this.bookStore = JSON.parse(localStorage.getItem('singleBook')) || [];
+  }
+
+  getBookList() {
```
Comment (TedLivist) [inline modules/books.js]: - [x] In keeping with the ES6 syntax requirement, kindly refactor this function to be an arrow function, like so:

  ```
  getBookList = () => {
    ...
  ```

File: modules/books.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    );
+    const Allbtn = document.querySelectorAll('.remove-button');
+    Allbtn.forEach((btn) => {
+      btn.addEventListener('click', () => {
+        this.deleteBook(btn.id);
+      });
+    });
+  }
+
+  addNewBook(title, author) {
```
Comment (TedLivist) [inline modules/books.js]: - [x] Kindly refactor to an arrow function

File: modules/books.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    const addbook = {
+      title,
+      author,
+      id,
+    };
+    this.bookStore.push(addbook); localStorage.setItem('singleBook', JSON.stringify(this.bookStore));
+    this.getBookList();
+  }
+
+  deleteBook(id) {
```
Comment (TedLivist) [inline modules/books.js]: - [x] Kindly refactor to an arrow function.

File: index.html [hunk 4]
```diff
@@ -1,13 +1,58 @@
...
+        <h2>Awesome Books</h2>
+        <ul>
+            <li><a id="list-link" href="#">List</a></li>
+            <li><a id="add-link" href="#">Add new</a></li>
+            <li><a id ="contact-link" href="#">Contact</a></li>
+        </ul>
+    </nav>
+    <div class="date">
+        <p>April 29th 2022</p>
+    </div>
```
Comment (TedLivist) [inline index.html]: - [x] Your date and time should be displayed using the `luxon` package. Kindly do this.

File: index.js [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+const booksLibrary = new Book();
+booksLibrary.getBookList();
+const formData = document.getElementById('new-book');
+formData.addEventListener('submit', (e) => {
+  e.preventDefault();
+  const getTitle = document.getElementById('title').value;
+  const getAuthor = document.getElementById('author').value;
+  booksLibrary.addNewBook(getTitle, getAuthor);
+});
```
Comment (TedLivist) [inline index.js]: - [x] **[OPTIONAL]** For a better user experience, it would be nice to clear the inputs after submission.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @bizip,

Good job so far!
✔️ Good documentation
✔️ Toggle of display
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the **[OPTIONAL]** prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: modules/books.js [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+    getBookList = () => {
+      const bookList = document.getElementById('book-list');
+      bookList.innerHTML = '';
+      bookList.innerHTML += this.bookStore.map(
+        (el, index) => ` <div class="${index % 2 === 0 ? 'dark' : 'mediumdark'}">
+          <h4>${el.title}</h4>
+          <p>${el.author}</p>
+          <button type="button" id=${el.id} class='remove-button'>Remove</button>
+          </div>`,
+      );
```
Comment (elmar8287) [inline modules/books.js:15]: - [ ] OPTIONAL - Please, consider that this `comma` is not needed - try to delete it from line 14, after the closing `div` tag - your UI will look more professional.
Screen Shot 2022-05-03 at 23 22 37

**Review by elmar8287 — APPROVED** [review]
Hi, @bizip.

# APPROVED  👍 🥇 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

Congratulations! 🎉 



## Please, consider some OPTIONAL points before merging 👍 

- [ ] Find OPTIONAL comment. under the review
- [ ] Please, try to fix the broken link to the MIT.md file.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: modules/books.js (hunk 6 above)
Comment (bizip) [reply modules/books.js:15]: Ohh thank you, I am going to do that

**Pull request merged**